core/tests/exhaustive_sync/experiment.rs

```diff
@@ -88,7 +88,8 @@ impl Experiment {
                 .spawn(move || loop {
                     match Self::grab_ready_trial_for_thread(thread_id, thread_state.clone()) {
                         (Some(mut work), _) => {
-                            let mutants = work.execute(thread_id);
+                            let mut mutants = work.execute(thread_id);
+                            mutants.reverse();
```

Comment:
Because mutants are pushed and popped, the last-pushed mutant was being run first. That means the first trials start with many SyncAndChecks (last mutation added by get_mutants), which are the most expensive and least useful operation for rapid feedback on fuzzer changes